docs/markdown/Tutorial.md

```diff
@@ -15,7 +15,7 @@ Debian-derived systems such as Ubuntu they can be installed with the
 following command:
 
 ```
-sudo apt install libgtk-3-dev
+sudo apt install libgtk-4-dev
```

Comment:
Wouldn't you also need to install gtksourceview here?

Reply:
Since this library is unused in the tutorial and only serves as an example of how to add multiple dependencies, I suggest adding it as a comment in meson.build later in the article where it is relevant:

```meson
project('tutorial', 'c')
gtkdep = dependency('gtk4')

# Make sure to install the new dependency first with
# sudo apt install libgtksourceview-5-dev
gtksourceview_dep = dependency('gtksourceview-5')
executable('demo', 'main.c', dependencies : [gtkdep, gtksourceview_dep])
```